components/input/input.html

```diff
@@ -134,5 +134,59 @@ <h3 class="comp-lib-h2">Input Search box</h3>
         
         </div>
     </div>
+
+    <h3 class="comp-lib-h2">Check Box</h3>
+    <div class="component_demo_container">
+    <div>
+        <!-- Checkbox Component Starts -->
+        <label class="dui-inp-chkbox dui-util-txt-sm">Checkbox 101
```

Comment:
You could use for attributes in label matching the id attribute in input for accessibility

Reply:
I left it out for the user, most probably will be making it part of post MVP.
Thanks.